src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -100,95 +100,84 @@ class UniversalNavbar extends React.Component {
     )
 
     return (
-      <div style={{ height: 64 }}>
```

Comment:
@iamvictorli any idea why we had this extra div?

Reply:
No I don't. I don't think this is needed as there is another div that has a height.